com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariableBase.cs

```diff
@@ -79,6 +79,10 @@ public void Initialize(NetworkBehaviour networkBehaviour)
         public virtual void SetDirty(bool isDirty)
         {
             m_IsDirty = isDirty;
```

Comment:
Should probably remove protected from m_IsDirty so this function isn't accidentally bypassed in the future.

Reply:
Yeah, if this is not consider an API breakage, definitely.

Reply:
protected is API change/breakage... :(